Views/Form/Login.cshtml

```diff
@@ -0,0 +1,39 @@
+@model NewPlayer
+@using registrationForm.Controllers;
+
+@{
+    ViewData["Title"] = "Register";
+}
+
+<h1>Register</h1>
+
+@using Html.BeginForm("Login", "Form", FormMethod.Post))
+{
+    <div>
+
+    @* NOTE: First Name: @Html.TextBoxFor(m => m.PlayerFirstName) *@
+
+    First Name:
+    <br>
```

Comment:
`<br>` elements can be hard to re-style down the road.  If you want a new line, better to wrap your text in a `<div>`.  Because that's a block level element, it'll create a new line, but also be easier to style if we need to change things up.